From 92df370621b6e1286ef273310ad47371456a5cf0 Mon Sep 17 00:00:00 2001
From: Baptiste Assmann <bedis9@gmail.com>
Date: Tue, 24 Jun 2014 11:10:00 +0200
Subject: [PATCH 09/18] BUG/MINOR: config: http-request replace-header arg typo

http-request replace-header was introduced with a typo which prevents it
to be conditionned by an ACL.
This patch fixes this issue.
---
 src/proto_http.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/proto_http.c b/src/proto_http.c
index 878951f..231d49a 100644
--- a/src/proto_http.c
+++ b/src/proto_http.c
@@ -8923,7 +8923,7 @@ struct http_req_rule *parse_http_req_cond(const char **args, const char *file, i
 		cur_arg = 1;
 
 		if (!*args[cur_arg] || !*args[cur_arg+1] || !*args[cur_arg+2] ||
-		    (*args[cur_arg+3] && strcmp(args[cur_arg+2], "if") != 0 && strcmp(args[cur_arg+2], "unless") != 0)) {
+		    (*args[cur_arg+3] && strcmp(args[cur_arg+3], "if") != 0 && strcmp(args[cur_arg+3], "unless") != 0)) {
 			Alert("parsing [%s:%d]: 'http-request %s' expects exactly 3 arguments.\n",
 			      file, linenum, args[0]);
 			goto out_err;
-- 
1.8.5.5

